docs/source/developer/extension_migration_2_3.rst

```diff
@@ -25,18 +25,19 @@ Upgrading library versions using the upgrade script
 
 JupyterLab 3.0 provides a script to upgrade an existing extension to use the new extension system and packaging.
 
-First, make sure to update to JupyterLab 3.0 with ``pip``:
+First, make sure to update to JupyterLab 3.0 and install ``jupyter-packaging`` and ``cookiecutter``. With ``pip``:
 
 .. code:: bash
 
    pip install jupyterlab -U
+   pip install jupyter_packaging cookiecutter
```

Comment:
This is really confusing when we use a dash instead of an underscore in a package name. I really try to avoid it when I make a new package.

Reply:
To cut down on the confusion, we can use the dash when installing (but still have to use an underscore when importing)

```suggestion
   pip install jupyter-packaging cookiecutter
```